gdcdictionary/schemas/_terms.yaml

```diff
@@ -5584,7 +5584,7 @@ marijuana_use_per_week:
       cde_version:
       term_url:
 
-yes:
+"yes":
```

Comment:
Note to other reviewers:  These two changes to _terms.yaml have already been merged into develop to prevent the other PRs from having issues.